packages/clima_domain/lib/repos/forecasts_repo.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:clima_core/failure.dart';
+import 'package:clima_domain/entities/city.dart';
+import 'package:clima_domain/entities/forecasts.dart';
+import 'package:dartz/dartz.dart';
+import 'package:riverpod/riverpod.dart';
+
+abstract class ForcastsRepo {
+  Future<Either<Failure, Forecasts>> getForcasts(City city);
+}
+
+final weatherRepoProvider =
```

Comment:
This should be renamed to `forecastsRepoProvider`. It should also be overriden with `forecastsRepoImplProvider` in `clima_ui/lib/main.dart` (see https://github.com/CentaurusApps/clima/blob/master/packages/clima_ui/lib/main.dart#L20 for an example).